### PR TITLE
[Scope change] WIP batch import of tags

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: AB#FILL_IN_YOUR_ISSUE_ID

## Checklist:

-   [ ] I have performed a self-review of my own code.
-   [ ] I have linked my DevOps task using the AB# tag.
-   [ ] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/apps/ScopeChangeRequest/Components/AdvancedDocumentSearch/AdvancedDocumentSearch.tsx [hunk 1]
```diff
@@ -3,18 +3,21 @@ import { Button, Icon, Progress, Scrim, SingleSelect, TextField } from '@equinor
...
-import { getDocumentsByTag } from '../../api/STID/getDocumentsByTag';
 import { Result } from './Results';
 import { SubResults } from './SubResult';
 import { AdvancedSearch, ModalHeader, Wrapper, Title, SearchField } from './advancedSearch.styles';
 import { ProcoSysTypes } from '../../types/ProCoSys/ProCoSysTypes';
 import { useReferencesSearch } from '../../hooks/Search/useReferencesSearch';
 import { useCancellationToken } from '@equinor/hooks';
+import { Switch } from '../../../../components/JSXSwitch/Components/Switch';
+import { Case } from '@equinor/JSX-Switch';
+import { httpClient } from '../../../../Core/Client/Functions';
```
Comment (Noggling) [inline src/apps/ScopeChangeRequest/Components/AdvancedDocumentSearch/AdvancedDocumentSearch.tsx:14]: alias

File: src/apps/ScopeChangeRequest/api/PCS/Batch/batchTag.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { httpClient } from '../../../../../Core/Client/Functions';
```
Comment (jaysencpp) [inline src/apps/ScopeChangeRequest/api/PCS/Batch/batchTag.ts:1]: alias import

File: src/apps/ScopeChangeRequest/Components/AdvancedDocumentSearch/NotFoundList.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+export const ErrorContainer = styled.div`
+    min-width: 250px;
+    min-height: 15px;
+    width: -webkit-fill-available;
+    height: auto;
+    border-radius: 5px;
+    background-color: ${tokens.colors.ui.background__danger.hex};
+    display: flex;
+    padding: 0.5em 0.5em;
+    display: flex;
```
Comment (jaysencpp) [inline src/apps/ScopeChangeRequest/Components/AdvancedDocumentSearch/NotFoundList.tsx:44]: display: flex to ganger  her

File: src/apps/ScopeChangeRequest/Components/AdvancedDocumentSearch/AdvancedDocumentSearch.tsx [hunk 4]
```diff
@@ -192,54 +189,106 @@ export const AdvancedDocumentSearch = ({
...
-                                        )}
-                                    </>
+                            <Switch
+                                defaultCase={
+                                    <TextField
+                                        id={'Stid document selector'}
+                                        value={searchText}
+                                        inputIcon={
+                                            <>
+                                                {isLoading ? (
```
Comment (jaysencpp) [inline src/apps/ScopeChangeRequest/Components/AdvancedDocumentSearch/AdvancedDocumentSearch.tsx:199]: er det verdt å bruke Switch componenten her også?

Comment (Gustav-Eikaas) [reply src/apps/ScopeChangeRequest/Components/AdvancedDocumentSearch/AdvancedDocumentSearch.tsx:199]: Ja

**Review by jaysencpp — APPROVED** [review]

**Pull request merged**